qemu/tests/cfg/nvdimm.cfg

```diff
@@ -39,7 +39,12 @@
     share_mem = yes
     nv_file = "${mount_dir}/nv0"
     depends_pkgs = "ndctl "
-
+    RHEL.9:
+        repo_install_cmd = "dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm"
+    RHEL.8:
+        repo_install_cmd = "dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm"
+    RHEL.7:
+        repo_install_cmd = "yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm"
```

Comment:
Is this repo needed by `cmake` package only ?

Reply:
@yanan-fu 
It's needed by pandoc mandatory for compiling the project and it's only available in the epel repo, for cmake it's not needed.

Reply:
Okay, i thought it is needed by the new pkg `cmake` before.  It is strange the repo is needed for the existing pkg.

I just checked, those pkgs all are in RHEL and also fedora compose (I did not check all Linux).  Did you hit any error without the epel repo ？  Could you share more details ? Thanks!

Reply:
@yanan-fu 
Sure, I've updated the bz with the [debug.log](https://bugzilla.redhat.com/show_bug.cgi?id=2178166#c2) obtained before the changes (only the "depends_pkgs" fix)

There you'll be able to see:
```bash
[stdlog] 2023-03-31 07:36:20,590 avocado.virttest.utils_package ERROR| 'yum install -y pandoc' execution failed with Updating Subscription Management repositories.
[stdlog] Unable to read consumer identity
[stdlog] 
[stdlog] This system is not registered with an entitlement server. You can use subscription-manager to register.
[stdlog] 
[stdlog] Last metadata expiration check: 0:00:03 ago on Fri 31 Mar 2023 07:36:12 PM CST.
[stdlog] No match for argument: pandoc
[stdlog] Error: Unable to find a match: pandoc
```

Reply:
@yanan-fu FYI from [pandoc org web page](https://pandoc.org/installing.html)

`
Check whether the pandoc version in your package manager is not outdated. Pandoc is in the Debian, Ubuntu, Slackware, Arch, Fedora, NiXOS, openSUSE, gentoo and Void repositories.
`

Reply:
As the discussion offline,  the `pandoc` was removed from RHEL9 compose but still needed by this test.  So let's apply this change, thanks!